server/routes/categories.js

```diff
@@ -19,8 +19,8 @@ router.get('/automotive', (req, res) => {
 
   return new Post()
     .where({ category_id: 1 })
-    .fetch({
-      withRelated: ['customerId', 'postStatusId', 'postPriorityId']
+    .fetchAll({
```

Comment:
Does using fetchAll here along with the .where above, return only posts who's category_id matches 1?

Reply:
Yes, I tested it both ways, with just fetch it will only return 1 post but if I use fetchAll then it will return all posts within that category id